tests_requre/service/conftest.py

```diff
@@ -1,122 +1,50 @@
-    "status": "success",
-    "status_per_chroot": {"fedora-43-x86_64": "success", "fedora-42-x86_64": "pending"},
-    "chroots": ["fedora-43-x86_64", "fedora-42-x86_64"],
-    "commit_sha": "80201a74d96c",
-    "web_url": "https://copr.something.somewhere/270520",
-    "srpm_logs": "Some boring logs.",
-    "ref": "80201a74d96c",
-    "repo_namespace": "marvel",
-    "repo_name": "aos",
-    "git_repo": "https://github.com/marvel/aos",
-    "https_url": "https://github.com/marvel/aos.git",
-    "pr_id": 2705,
-}
-
-
-@pytest.fixture()
-def pr_model():
-    yield PullRequestModel.get_or_create(
-        pr_id=build_info_dict["pr_id"],
-        namespace=build_info_dict["repo_namespace"],
-        repo_name=build_info_dict["repo_name"],
-        project_url=build_info_dict["git_repo"],
-    )
+import pytest
 
+from packit_service.service.app import application
 
-@pytest.fixture()
-def different_pr_model():
-    yield PullRequestModel.get_or_create(
-        pr_id=4,
-        namespace="the-namespace",
-        repo_name="the-repo-name",
-        project_url="https://github.com/the-namespace/the-repo-name",
-    )
 
+@pytest.fixture
+def client():
+    application.config["TESTING"] = True
+    # this affects all tests actually, heads up!
+    application.config["SERVER_NAME"] = "localhost:5000"
+    application.config["PREFERRED_URL_SCHEME"] = "http"
 
-@pytest.fixture()
-def multiple_copr_builds(pr_model, different_pr_model):
-    srpm_build = SRPMBuildModel.create(build_info_dict["srpm_logs"])
-    yield [
-        CoprBuildModel.get_or_create(
-            build_id=build_info_dict["build_id"],
-            commit_sha=build_info_dict["ref"],
-            project_name=build_info_dict["project"],
-            owner=build_info_dict["owner"],
-            web_url=build_info_dict["web_url"],
-            target="fedora-43-x86_64",
-            status="success",
-            srpm_build=srpm_build,
-            trigger_model=pr_model,
-        ),
-        CoprBuildModel.get_or_create(
-            build_id=build_info_dict["build_id"],
-            commit_sha=build_info_dict["ref"],
-            project_name=build_info_dict["project"],
-            owner=build_info_dict["owner"],
-            web_url=build_info_dict["web_url"],
-            target="fedora-42-x86_64",
-            status="pending",
-            srpm_build=srpm_build,
-            trigger_model=pr_model,
-        ),
-        CoprBuildModel.get_or_create(
-            build_id="123456",
-            commit_sha="687abc76d67d",
-            project_name="SomeUser-hello-world-9",
-            owner="packit",
-            web_url="https://copr.something.somewhere/123456",
-            target="fedora-43-x86_64",
-            status="success",
-            srpm_build=srpm_build,
-            trigger_model=different_pr_model,
-        ),
-    ]
+    with application.test_client() as client:
+        # the first call usually fails
+        client.get("http://localhost:5000/api/")
```

Comment:
This line "solves" the problem with the fixture. I'll take a look for better solution later to not block this PR.